src/Compilers/CSharp/CscCore/project.json

```diff
@@ -1,27 +1,35 @@
-{
+﻿{
   "dependencies": {
     "Microsoft.NETCore.Platforms": "1.0.0",
-    "System.Collections": "4.0.10",
-    "System.Console": "4.0.0-beta-23123",
-    "System.Diagnostics.Debug": "4.0.10",
-    "System.Globalization": "4.0.10",
-    "System.IO.FileSystem": "4.0.0",
-    "System.IO.FileSystem.Primitives": "4.0.0",
-    "System.Reflection": "4.0.10",
-    "System.Runtime": "4.0.20",
-    "System.Runtime.InteropServices": "4.0.20",
-    "System.Runtime.Extensions": "4.0.10",
-    "System.Text.Encoding": "4.0.10",
-    "System.Threading": "4.0.10",
-    "System.Threading.Tasks": "4.0.10",
-    "System.Threading.Thread": "4.0.0-beta-23123",
-    "System.Xml.XDocument": "4.0.10",
-    "System.Xml.XmlDocument": "4.0.0",
-    "System.IO.Pipes": "4.0.0-beta-23123",
-    "System.Diagnostics.Process": "4.0.0-beta-23123",
-    "System.AppContext": "4.0.0",
-    "System.Security.Cryptography.Hashing.Algorithms": "4.0.0-beta-23123",
-    "System.Linq": "4.0.0"
+    "Microsoft.NETCore.Runtime.CoreCLR": "1.0.1-beta-23310",
+    "System.AppContext": "4.0.1-beta-23310",
+    "System.Collections": "4.0.11-beta-23310",
+    "System.Console": "4.0.0-beta-23302",
```

Comment:
@weshaggard Definitely. Right now those versions are unimportant since the runtime assemblies are copied over from a zip file right now (and overwrite the ones NuGet gives), but this will be updated to use the NuGet runtime assemblies ASAP.